packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -353,16 +353,16 @@ export function useSelectableCollection(options: AriaSelectableCollectionOptions
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  // If not virtualized, scroll the focused element into view when the focusedKey changes.
-  // When virtualized, Virtualizer handles this internally.
   useEffect(() => {
+    // If not virtualized, scroll the focused element into view when the focusedKey changes.
+    // When virtualized, Virtualizer handles this internally.
     if (!isVirtualized && manager.focusedKey && scrollRef?.current) {
       let element = scrollRef.current.querySelector(`[data-key="${manager.focusedKey}"]`) as HTMLElement;
       if (element) {
         scrollIntoView(scrollRef.current, element);
       }
     }
-  }, [isVirtualized, scrollRef, manager.focusedKey]);
+  }, [isVirtualized, scrollRef, manager, delegate, selectOnFocus]);
```

Comment:
what is this change specifically for?

Reply:
hmm not sure, this seems wrong. Maybe rob remembers. But looks like maybe a merge issue... let me try reverting

Reply:
i do not remember, unfortunately, looks like a merge issue though given that those are not values used in the effect

looks like you were able to revert it successfully